.gitlab/ci/on-push.yml

```diff
@@ -1,8 +1,14 @@
+# This rule is to not run the build for docker update branches (for non-nightly packs)
+.filter-non-nightly-docker-updates-rule:
+  rules:
+    - if: '$CI_COMMIT_BRANCH =~ /^demisto\// && $CI_COMMIT_BRANCH !~ /^demisto\/.*-nightly$/'
+      when: never
```

Comment:
Can you explain this if statement in words?

Reply:
@Shellyber Don't forget this

Reply:
Yes, so basically a branch name that starts with demisto/ but does not include the `-nightly` in the branch.
Few examples:
The regex will catch:
- demisto/python3
- demisto/google-test
- demisto/api3-something

Will not catch:
- demisto/python3-nigthly
- demisto/something-nightly
- any-regular-branch-name